api/main.go

```diff
@@ -142,11 +154,14 @@ func main() {
 			Page    int    `query:"page"`
 			PerPage int    `query:"per_page"`
 			Filter  string `query:"filter"`
+			Pending int    `query:"pending"`
```

Comment:
Why this is not a boolean? Using an integer here seems wrong. What `pending` as 25 means?